pylint/config/find_default_config_files.py

```diff
@@ -29,54 +33,52 @@ def _toml_has_config(path):
     return True
 
 
-def _cfg_has_config(path):
+def _cfg_has_config(path: Path) -> bool:
     parser = configparser.ConfigParser()
     parser.read(path, encoding="utf-8")
     return any(section.startswith("pylint.") for section in parser.sections())
```

Comment:
@Pierre-Sassoulas I have not copied the handling of malformed `setup.cfg`. If it is malformed we should raise a warning like we do for `.toml` so we should probably handle that in another place where we have access to the `linter` object.

Reply:
It's inside a function that check if the setup.cfg contains a pylint configuration, not handling it means a malformed / unused setup.cfg would crash pylint. Could we discuss and merge the other MR first ?